internal/cmd/config/get.go

```diff
@@ -0,0 +1,60 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/hetznercloud/cli/internal/cmd/util"
+	"github.com/hetznercloud/cli/internal/state"
+	"github.com/hetznercloud/cli/internal/state/config"
+)
+
+func NewGetCommand(s state.State) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "get <key>",
+		Short:                 "Get a configuration value",
+		Long:                  "Get a configuration value. For a list of all available configuration options, run 'hcloud help config'.",
+		Args:                  util.Validate,
+		TraverseChildren:      true,
+		DisableFlagsInUseLine: true,
+		SilenceUsage:          true,
+		RunE:                  state.Wrap(s, runGet),
+	}
+	cmd.Flags().Bool("global", false, "Get the value globally")
+	cmd.Flags().Bool("allow-sensitive", false, "Allow showing sensitive values")
+	return cmd
+}
+
+func runGet(s state.State, cmd *cobra.Command, args []string) error {
+	global, _ := cmd.Flags().GetBool("global")
+	allowSensitive, _ := cmd.Flags().GetBool("allow-sensitive")
+
+	if global {
+		// set context to nil and then reload
+		config.OptionContext.OverrideAny(s.Config(), nil)
+		s.Config().Reset()
+		if err := config.ReadConfig(s.Config(), nil); err != nil {
+			return err
+		}
```

Comment:
This code is duplicated between `get.go` and `list.go`. I think this makes more sense to have in `state/config/config.go`. That way `config.Reset()` could also be dropped from the interface.